src/Navbar.test.js

```diff
@@ -41,28 +41,25 @@ describe("Navbar", () => {
     expect(wrapper).toMatchSnapshot();
     expect(wrapper.find(".register").length).toBe(0);
     expect(wrapper.find(".login").length).toBe(0);
-    expect(wrapper.find(".profilePage").get(0).props.children).toBe("Profile");
-    expect(wrapper.find(".logout").get(0).props.children).toBe("Logout");
-    expect(wrapper.find(".addConference").get(0).props.children).toBe(
-      "Add Conference"
-    );
+    expect(findElement(".profilePage")).toBe("Profile");
+    expect(findElement(".logout")).toBe("Logout");
+    expect(findElement(".addConference")).toBe("Add Conference");
```

Comment:
not sure what this is doing, do you mean the text is "Add Conference"? why is the element equal to be a string?

Reply:
findElement is a method which simplifies `wrapper.find(".addConference").get(0).props.children`